src/modules/authentication/AuthenticationWrapper.ts

```diff
@@ -180,6 +200,15 @@ export class AuthenticationWrapper implements IAuthenticationWrapper {
     }
   }
 
+  private getClaims(): string | undefined {
+    const account = this.getAccount();
+    if(account && (this.sampleQuery.sampleUrl !== '')){
```

Comment:
Should we check the existence of sampleQuery before evaluating whether sampleUrl is empty?

Reply:
Like this?
if(account && this.sampleQuery && this.sampleQuery.sampleUrl)

Reply:
We don't have to because sampleQuery is an instance variable of the AuthenticationWrapper class